task-management/src/main/webapp/vue-app/tasks-management/components/ProjectTasks/TasksViewHeaderStatus.vue

```diff
@@ -143,7 +143,21 @@ export default {
     },
     limitTasksToshow() {
       return this.tasksStatsStartValue;
-    }
+    },
+    taskStatusLabel() {
+      switch (this.status && this.status.name) {
```

Comment:
you may do it this way IMO to make it more dynamic :

```javascript
const key = `exo.tasks.status.${this.status?.name.toLowerCase()}`;
const translatedLabel = this.$t(key);
const label = translatedLabel === key && this.status?.name || translatedLabel;
```

Reply:
I fixed it